llvm/test/CodeGen/ARM/big-endian-vector-trunc.ll

```diff
@@ -0,0 +1,31 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=armebv7-unknown-none-eabihf -mattr=+neon < %s | FileCheck %s
```

Comment:
is there any existing ARM codegen test file that does something similar that we're just missing BE test coverage?

Reply:
I've had a search and can't find any existing tests which cover this. The only way I've been able to hit this code path is with operations on `<8 x i64>`, which is very over-sized for NEON so gets expanded multiple times.

Reply:
OK - please rename this vector-trunc.ll and add equivalent LE test checks as well.

Reply:
Done